docs/standard/net-standard.md

```diff
@@ -29,7 +29,7 @@ To find the highest version of .NET Standard that you can target, do the followi
 
 1. Find the row that indicates the .NET implementation you want to run on.
 2. Find the column in that row that indicates your version starting from right to left.
-3. The column header indicates the .NET Standard version that your target supports (and any lower .NET Standard versions will also support it).
+3. The column header indicates the .NET Standard version that your target supports. You may also target any lower .NET Standard version. Higher .NET Standard versions will also support your implementation.
```

Comment:
How about:

```
The column header indicates the .NET Standard version that your chosen .NET implementation version supports. You may also target lower .NET Standard versions.
```

This part is true: `Higher .NET Standard versions will support your implementation.` I think it is almost an orthogonal topic, for two different reasons: compatibility is primary a function of the .NET implementation and less the standard and what the future looks like is a policy issue that isn't represented by this table.